.gitignore

```diff
@@ -42,3 +42,6 @@ index.js
 nohup.out
 
 site/
+
+derby.log
+metastore_db/
```

Comment:
What are these files?

Reply:
Those are (I believe) created by Jacoco when you run tests.